private/underlay/conn/conn.go

```diff
@@ -398,38 +394,35 @@ func (cc *connUDPBase) initConnUDP(network string, laddr, raddr *net.UDPAddr, cf
 	return nil
 }
 
-// @ preserves c.Mem()
+// @ preserves acc(c.Mem(), _)
```

Comment:
A bit of a nitpick but I would change this to requires (and same for similar cases below). I find it a bit confusing since it is not really preserved.

Reply:
I was gonna change this, but decided against. If one expands the definition of `_`, I would argue that it is preserved:
```
preserves exists p perm :: 0 < p && p < perm(c.Mem()) && acc(c.Mem(), p)
```